phpunit.xml.dist

```diff
@@ -39,6 +39,10 @@
         <!-- env name="TRUSTED_HOSTS" value="'^localhost|example\.com$'" -->
         <!-- ###- symfony/framework-bundle ### -->
 
+        <env name="BITBUCKET_AUTH_METHOD" value="http_password"/>
+        <env name="BITBUCKET_TOKEN" value="alessandro_minoccheri"/>
+        <env name="BITBUCKET_SECRET" value="pL4gSBDh2Bc4VZNvRrdm"/>
```

Comment:
please don't publish your token here, but put this on `.env.test.local`! no?

Reply:
Ouch!